style.css

```diff
@@ -49,7 +53,72 @@ main {
 
 /* Navigation stylings */
 
+.header-body {
+  background-color: #000;
+}
+
 .ul-menu {
   list-style-type: none;
   padding: 0;
 }
```

Comment:
Sorry for not seeing it in the previous PR, but using a specific element name in your class name is not the best approach. What if another element would be more fitting, and you'll change it, the name would make less sense, `menu` would be sufficient imo :)